### ISSUE TITLE
Devitalia: error while downloading software catalog stats

### ISSUE THREAD

**Issue by LucaPrete** [teamdigitale/dpt-services-dashboard-scripts#6]
While running the devitalia ingestion process, we get:

```
2020-04-07 10:13:22,532 - ganalytics - INFO - Already authenticated, skip auth
Traceback (most recent call last):
  File "/opt/ingestion_scripts/devitalia/fetch-engines/main.py", line 56, in <module>
    compute_stats(args)
  File "/opt/ingestion_scripts/devitalia/fetch-engines/main.py", line 15, in compute_stats
    stats = engine.compute_stats()
  File "/opt/ingestion_scripts/devitalia/fetch-engines/engines/engine.py", line 71, in compute_stats
    method_to_call()
  File "/opt/ingestion_scripts/devitalia/fetch-engines/engines/onboarding_stats.py", line 71, in num_pas_with_softwares
    self._get_softwares()
  File "/opt/ingestion_scripts/devitalia/fetch-engines/engines/onboarding_stats.py", line 48, in _get_softwares
    sws = yaml.safe_load(sws)
  File "/usr/lib/python3/dist-packages/yaml/__init__.py", line 94, in safe_load
    return load(stream, SafeLoader)
  File "/usr/lib/python3/dist-packages/yaml/__init__.py", line 72, in load
    return loader.get_single_data()
  File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 35, in get_single_data
    node = self.get_single_node()
  File "/usr/lib/python3/dist-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/usr/lib/python3/dist-packages/yaml/composer.py", line 58, in compose_document
    self.get_event()
  File "/usr/lib/python3/dist-packages/yaml/parser.py", line 118, in get_event
    self.current_event = self.state()
  File "/usr/lib/python3/dist-packages/yaml/parser.py", line 193, in parse_document_end
    token = self.peek_token()
  File "/usr/lib/python3/dist-packages/yaml/scanner.py", line 128, in peek_token
    self.fetch_more_tokens()
  File "/usr/lib/python3/dist-packages/yaml/scanner.py", line 220, in fetch_more_tokens
    return self.fetch_value()
  File "/usr/lib/python3/dist-packages/yaml/scanner.py", line 576, in fetch_value
    self.get_mark())
yaml.scanner.ScannerError: mapping values are not allowed here
  in "<byte string>", line 9, column 41:
     ... ng="it" class="no-js" prefix="og: http://ogp.me/ns#">
```

It seems related to syntax/format errors in `https://onboarding.developers.italia.it/repo-list`, but putting its content in a yaml linter, everything seems to be correct.

**Comment by LucaPrete** [comment]
Update: the issue seems not realted to the onboarding website, but to https://crawler.developers.italia.it/softwares.yml, which looks to be invalid

**Comment by sebbalex** [comment]
This should be considered closed by 8351f37 @LucaPrete can you confirm that?

**Comment by LucaPrete** [comment]
Yes! Let's close it ;)
